src/components/shared/Typography.js

```diff
@@ -8,6 +8,7 @@ export const Title = styled.h1`
 export const SubTitle = styled.p`
   font-weight: normal;
   font-size: ${props => (props.size ? props.size : '1.5rem')}
+  margin-left: ${props => (props.marginLeft ? props.marginLeft : '0rem')}
```

Comment:
Make it: `margin-left: ${props => (props.marginLeft ? props.marginLeft + 'rem' : '0rem')}`

Reply:
@fragm3 Why is this change required ?

Reply:
@harsh-jindal when calling would be like: `<SubTitle marginLeft={1.4}>`, which is better than what we are doing right now.